react-instantsearch-hooks/query-suggestions/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Most of it is generated by `tsc --init`.

I added the JSX transform so we don't have to import React.